README.md

```diff
@@ -1,77 +1,65 @@
 # ![nf-core/cageseq](docs/images/nf-core-cageseq_logo.png)
 
-**CAGE-seq pipeline**.
+Nextflow CAGE-seq analysis pipeline, part of the nf-core community
```

Comment:
I think the fact that it's part of the nf-core community is already obvious.
I would go for only `CAGE-seq analysis pipeline`

Reply:
Looks like not all pipelines have that subtitle (RNA-seq, ATAC-seq). I will remove it, because the github description field should already fulfill that function.